project/Dependencies.scala

```diff
@@ -88,6 +88,8 @@ object Dependencies {
     "org.mongodb" %% "casbah" % "3.0.0"
   )
 
+  val sparkBackendDependencies = List("org.json4s" %% "json4s-jackson" % "3.4.0")
```

Comment:
There are a lot of reasons why I might want to use a diff library here and there. The most important reason is "that's what's being used everywhere else". 

Aside: If we were to switch our json handling libraries across the board I'd want it to be circe, which hasn't even been part of this discussion yet.